src/ClientApp/src/app/polling-station-search/polling-station-search.component.ts

```diff
@@ -108,13 +107,21 @@ export class PollingStationSearchComponent implements OnInit, AfterViewInit, OnD
   }
 
   getPollingStationinfoBubble(group: PollingStationGroup): string {
-    // TODO: style info bubble text
-    return group.pollingStations.reduce((accumulator, currentValue)=>accumulator+'<br/>'+currentValue.pollingStationNumber,'');
+    return group.pollingStations.reduce((accumulator, currentValue) =>
+      accumulator + `<div style="width: 472px;">
+     <div style="font-family: Titillium Web;font-style: normal;font-weight: 600;font-size: 34px;line-height: 52px;display: flex;align-items: center;text-align: center;"> Sectia de votare ${currentValue.pollingStationNumber},  ${currentValue.locality}</div> 
```

Comment:
i tested locally and it works , almost perfect.
Please add `encapsulation: ViewEncapsulation.None` for this component and it will allow use of classes defined in scss.
After changing view encapsulation please extract inline style to css clases